### PR TITLE
docs: added a number of documents regarding the Twist Solid pipeline

### PR DESCRIPTION
**Pull request by jonca79**
### This PR:

(If this is a release PR, no need to add following. Leave this part empty)
(Use the following lines to create a PR text body. Make sure to remove all non-relevant one after you're done)
(Repeat each field as many times as necessary)

Added: for new features.
Changed: for changes in existing functionality.
Deprecated: for soon-to-be removed features.
Removed: for now removed features.
Fixed: for any bug fixes.
Security: in case of vulnerabilities.

### Review and tests:
- [ ] Tests pass
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Code review
- [ ] `CHANGELOG.md` is updated
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: docs/rules.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+#CONFIGSCHEMA__bcftools_id_snps#
+
+#### Resources settings (`resources.yaml`)
+
+#RESOURCESSCHEMA__bcftools_id_snps#
+
+---
+
+## call_small_cnv_deletions
```
Comment (Smeds) [inline docs/rules.md:27]: Add call small cnv amplifications?

File: docs/dna_fusions.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Fusion calling in DNA
```
Comment (Smeds) [inline docs/dna_fusions.md:1]: Update with fuseq wes information

File: docs/includes/abbreviations.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+*[HLA]: Human leukocyte antigen
+*[HRD]: Homologous recombination deficiency
+*[MAF]: Minor allele frequency
+*[MSI]: Microsatellite instability
+*[MSS]: Microsatellite stable 
+*[PoN]: Panel of normal
+*[QC]: Quality control
+*[RNA]: Ribonucleic acid
+*[sSNVs]: Synonymous single nucleotide variations
+*[nsSNVs]: Non-synonymous single nucleotide variations
```
Comment (Smeds) [inline docs/includes/abbreviations.md:20]:
```suggestion
*[nsSNV]: Non-synonymous single nucleotide variation
*[nsSNVs]: Non-synonymous single nucleotide variations
```

File: docs/dna_biomarkers.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+| nssnv_tmb_correction | 1.02 | Variant size times correction factor (correction factor = 1 / adjusted design size) |
+
+The main result is the TMB calculated using nsSNV only. However, TMB calculated using both nsSNVs and sSNVs are also provided as well as all the variants passing all filters.
+
+### Result file
+
+* `results/dna/tmb/{sample}_{type}.TMB.txt`
+
+## Microsatellite instability (MSI)
+To determine MSS or MSI status of the samples the percentage of sites that have microsatellite instability are calculated using **[MSIsensor-pro]([https://github.com/xjtu-omics/msisensor-pro])** v1.1.a. When the more than 10% of the sites are instable the sample is determined to have MSI status. The program uses a panel of normal to determine the normal level of instability in the used sites.
```
Comment (Smeds) [inline docs/dna_biomarkers.md:39]:
```suggestion
To determine MSS or MSI status of the samples the percentage of sites that have microsatellite instability are calculated using **[MSIsensor-pro]([https://github.com/xjtu-omics/msisensor-pro])** v1.1.a. When more than 10% of the sites are instable the sample is determined to have MSI status. The program uses a panel of normal to determine the normal level of instability in the used sites.
```

File: docs/overview.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  5.1 Calling using Mutect2 and Vardict  
+  5.2 Annotation using VEP and hydra-genetics  
+  5.3 Filtering using bcftools and hydra-genetics  
+6. **CNV**  
+  6.1 Calling using CNVkit and GATK CNV  
+  6.2 Merging using SVDB  
+  6.3 Annotation using SVDB and hydra-genetics  
+  6.4 Filtering using hydra-genetics  
+  6.5 CNV html report using hydra-genetics
+7. **Fusion** calling using GeneFuse
```
[the remark this answers is not in the document]
Comment (jonca79) [reply docs/overview.md]: done

File: docs/result_files.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+| `gvcf_dna/{sample}_{type}.mosdepth.g.vcf.gz` | g.vcf | Genomic vcf file with coverage in all position in the design |
+| **CNVs** | :~~: | ~~ |
+|  **`results/dna/cnv/{sample}_{type}/{sample}_{type}.pathology.cnv.html`** | html | Interactive html report of the filtered CNV results <br />using tumor content estimated by a pathologist |
+|  `results/dna/cnv/{sample}_{type}/{sample}_{type}.purecn.cnv.html` | html | Interactive html report of the filtered CNV results <br />using tumor content estimated by PureCN |
+|  **`results/dna/cnv/{sample}_{type}/{sample}_{type}.pathology.cnv_report.tsv`** | table | Table of filtered CNVs using tumor content <br />estimated by a pathologist |
+|  `results/dna/cnv/{sample}_{type}/{sample}_{type}.purecn.cnv_report.tsv` | table | Table of filtered CNVs using tumor content <br />estimated by PureCN |
+|  `results/dna/cnv/{sample}_{type}/{sample}_{type}.deletions.tsv` | table | Table of called small CNVs by in-house script |
+|  `results/dna/cnv/{sample}_{type}/{sample}_{type}.purecn.purity.txt` | table | Estimated tumor content and ploidity by PureCN |
+| **Fusions** | :~~: | ~~ |
```
[the remark this answers is not in the document]
Comment (jonca79) [reply docs/result_files.md:28]: done

File: docs/dna_fusions.md (hunk 2 above)
Comment (jonca79) [reply docs/dna_fusions.md:1]: done

File: workflow/schemas/rules.schema.yaml [hunk 7]
```diff
@@ -0,0 +1,250 @@
...
+            type: string
+            description: Copy number region data from GATK CNV
+          regions_file:
+            type: string
+            description: File with genes, gene regions with surrounding cnv-probes and the actual gene region
+      output:
+        type: object
+        description: list of outputs
+        properties:
+          deletions:
```
Comment (jonca79) [inline workflow/schemas/rules.schema.yaml:73]:
```suggestion
          deletion:
```

File: workflow/schemas/rules.schema.yaml [hunk 8]
```diff
@@ -0,0 +1,250 @@
...
+            type: string
+            description: Copy number region data from GATK CNV
+          regions_file:
+            type: string
+            description: File with genes, gene regions with surrounding cnv-probes and the actual gene region
+      output:
+        type: object
+        description: list of outputs
+        properties:
+          deletion:
```
Comment (jonca79) [inline workflow/schemas/rules.schema.yaml:73]:
```suggestion
          deletions:
```

File: workflow/schemas/rules.schema.yaml [hunk 9]
```diff
@@ -0,0 +1,250 @@
...
+            type: string
+            description: Copy number region data from GATK CNV
+          regions_file:
+            type: string
+            description: File with genes, gene regions with surrounding cnv-probes and the actual gene region
+      output:
+        type: object
+        description: list of outputs
+        properties:
+          amplification:
```
Comment (jonca79) [inline workflow/schemas/rules.schema.yaml:51]:
```suggestion
          amplifications:
```

**Review by Smeds — APPROVED** [review]

**Pull request merged**